ethereum/core/src/test/java/tech/pegasys/pantheon/ethereum/mainnet/precompiles/privacy/PrivacyPrecompiledContractTest.java

```diff
@@ -107,7 +107,7 @@ private PrivateTransactionProcessor mockPrivateTxProcessor() {
 
   private Enclave brokenMockEnclave() throws Exception {
     Enclave mockEnclave = mock(Enclave.class);
```

Comment:
throws Exception shouldn't be needed now it's throwing the runtime EnclaveException

Reply:
done.